app/utils/validators.js

```diff
@@ -79,6 +79,10 @@ export const validPhoneNumber = new RegExp(
   '^\\s*(?:\\+?(\\d{1,3}))?([-. (]*(\\d{3})[-. )]*)?((\\d{3})[-. ]*(\\d{2,4})(?:[-.x ]*(\\d+))?)\\s*$'
 );
 
+export const validEmail = new RegExp(
+  '[^@ \t\r\n]+@[^@ \t\r\n]+\.[^@ \t\r\n]+'
```

Comment:
Disable eslint `no-control-regex` error on this line

Reply:
Done